### PR TITLE
docs(elements): convert the `ng add` command to code-example

### PR DESCRIPTION
**Pull request by santoshyadavdev**
the command was hardly visible currently
changing it to shell for visibility

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/guide/elements.md [hunk 1]
```diff
@@ -119,7 +119,13 @@ The recently-developed [custom elements](https://developer.mozilla.org/en-US/doc
 
 In browsers that support Custom Elements natively, the specification requires developers use ES2015 classes to define Custom Elements - developers can opt-in to this by setting the `target: "es2015"` property in their project's [TypeScript configuration file](/guide/typescript-configuration). As Custom Element and ES2015 support may not be available in all browsers, developers can instead choose to use a polyfill to support older browsers and ES5 code.
 
-Use the [Angular CLI](cli) to automatically set up your project with the correct polyfill: `ng add @angular/elements --project=*your_project_name*`.
+Use the [Angular CLI](cli) to automatically set up your project with the correct polyfill: 
+<code-example language="bash">
```
Comment (gkalpak) [inline aio/content/guide/elements.md:123]:
```suggestion

<code-example language="sh">
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thank you, @santoshyadavdev!

You need to leave an empty line whenever you switch from Markdown to HTML, otherwise our Markdown parsers is thrown off.

File: aio/content/guide/elements.md (hunk 1 above)
Comment (santoshyadavdev) [reply aio/content/guide/elements.md]: Done, George. Thank you for your help 👍

**Review by gkalpak — APPROVED** [review]
LGTM :+1:

Could you please update the commit message to have a scope of `elements`. Maybe something like:

> docs(elements): convert the \`ng add\` command to code-example

**Comment by santoshyadavdev** [comment]
> LGTM 👍
> 
> Could you please update the commit message to have a scope of `elements`. Maybe something like:
> 
> > docs(elements): convert the `ng add` command to code-example

Done 👍

**Pull request closed without merging**